www/index.php

```diff
@@ -302,8 +302,8 @@ function renderTemplate($file, $params = [])
 $page->addScriptDeclaration("
 require(['jquery'], function($) {
     $(function() {
-        $('.moreOptions').hide();
-        $('#moreOptions').click(function() {
+        $('#moreOptions').hide();
+        $('#showMoreOptions').click(function() {
             var self = this;
             $('.moreOptions').slideDown('fast', function() {
```

Comment:
This selector now needs to be `$('#moreOptions')`

Reply:
Oops. Missed that. Will fix.